hr_attendance_hour_type_report/models/hr_attendance.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import datetime as dt
+import logging
+
+import pytz
+
+from odoo import _, api, exceptions, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    worked_hours_nighttime = fields.Float(
+        string="Night hours", compute="_compute_worked_hours_night", store=True
+    )
+    worked_hours_daytime = fields.Float(
+        string="Day hours",
+        compute="_compute_worked_hours_night",
+        store=True,
+        readonly=True,
+    )
+    date = fields.Date(
+        string="Date",
+        help="date of the attendance, from the payroll point of view",
+        compute="_compute_date",
+        store=True,
+    )
+    date_type = fields.Selection(
+        [("normal", "Weekday"), ("sunday", "Sunday"), ("holiday", "Public Holiday")],
+        compute="_compute_date_type",
+        store=True,
```

Comment:
```suggestion
        store=True,
        default="normal",
```

Reply:
it is a computed field, what is the benefit of having a default value?

Reply:
it can be useful if UI relies on this field and record is new() state so field initialized before compute method is trigered, change is not mandatory